server/engine/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -146,3 +146,4 @@ cobolParser.expectSpace=A blank is missing after ','
 #cics.exci.errormessage=encountered EXCI command without proper compiler directive
 procedureDivisionHeaderProcess.wrongNodeLocation=Operand '%s' was not defined in the LINKAGE SECTION
 procedureDivisionHeaderProcess.wrongDataName='%s' was not defined as a data-name
+variableNameCheck.notAllowedVariableName=The name %s was used for an item that was not defined as a data-name
```

Comment:
I think we should really tell the user that the conflicting item is a function declared in the repository. How about
```
The name %s cannot be used as a data-name because it was already declared as a function
```
?